app/models/picture.rb

```diff
@@ -9,15 +9,15 @@ class Picture < ApplicationRecord
 
   URL_ROOT          = Rails.root.join("public").to_s
   DEFAULT_DIRECTORY = File.join(URL_ROOT, "pictures")
-  Dir.mkdir(DEFAULT_DIRECTORY) unless File.directory?(DEFAULT_DIRECTORY)
+  FileUtils.mkdir_p(DEFAULT_DIRECTORY)
```

Comment:
Good find @isimluk.   Note, it feels wrong to be creating a directory when loading this class.  If the only read of `DEFAULT_DIRECTORY` is the `directory` class method, maybe it can be lazy created then.  But, I don't know if other places expect it there... oh well, follow up PR if you figure it out.